lib/smart_app_launch/smart_access_brands_suite.rb

```diff
@@ -0,0 +1,59 @@
+require_relative 'smart_access_brands_group'
+
+module SMARTAppLaunch
+  class SMARTAccessBrandsSuite < Inferno::TestSuite
+    id 'smart_access_brands'
+    title 'SMART Access Brands STU2.2'
+    version VERSION
+
+    description <<~DESCRIPTION
+      The SMART User-access Brands Test Suite verifies that Brand Bundle Publishers publish valid User-access
+      Brand Bundles according to the SMART App Launch IG
+      [User-access Brands and Endpoints](https://build.fhir.org/ig/HL7/smart-app-launch/brands.html#user-access-brands-and-endpoints)
```

Comment:
use the published url (not build.fhir.org) https://hl7.org/fhir/smart-app-launch/brands.html